music21/musicxml/xmlToM21.py

```diff
@@ -3861,17 +3861,32 @@ def xmlTechnicalToArticulation(self, mxObj):
             if tag in ('heel', 'toe'):
                 if mxObj.get('substitution') is not None:
                     tech.substitution = xmlObjects.yesNoToBoolean(mxObj.get('substitution'))
+            if tag == 'bend':
+                self.setBend(mxObj, tech)
             # TODO: <bend> attr: accelerate, beats, first-beat, last-beat, shape (4.0)
             # TODO: <bent> sub-elements: bend-alter, pre-bend, with-bar, release
             # TODO: musicxml 4: release sub-element as offset attribute
-
-
             self.setPlacement(mxObj, tech)
             return tech
         else:
             environLocal.printDebug(f'Cannot translate {tag} in {mxObj}.')
             return None
 
+    @staticmethod
+    def setBend(mxh, bend):
+        alter = mxh.find('bend-alter')
+        if alter is not None:
+            if alter.text is not None:
+                bend.bendAlter = interval.Interval(int(alter.text))
+        if mxh.find('pre-bend') is not None:
+            bend.preBend = True
+        if mxh.find('release') is not None:
+            try:
+                divisions = float(mxh.find('release').get('offset'))
+                bend.release = divisions / defaults.divisionsPerQuarter
```

Comment:
In writing to musicxml, music21 uses defaults.divisionsPerQuarter.  But in reading from musicxml, we need to use the divisions defined earlier in the score, which is in self.divisions on the measure parser.  

Write

```
bend.release = opFrac(divisions / defaults.divisionsPerQuarter)
```

opFrac will convert it to Fraction(1, 3) if it's released within a triplet, etc. and leave it alone as 0.5 if it's a regular eighth note, etc.